src/app/services/config/config.service.ts

```
@@ -37,34 +38,13 @@ const adminIP = 'https://amritwprdev.piramalswasthya.org/';
 @Injectable()
 export class ConfigService {
 
-    // private _commonBaseURL: String = `${commonIP}commonapi-v1.0/`;
-    private _commonBaseURL: String = `https://amritwprdev.piramalswasthya.org/commonapi-v1.0/`;
-    // private _commonBaseURL: String=`http://localhost:8080/`;
-    // private openCommonURL: String = `${commonIP}commonapi-v1.0/`;
-    private openCommonURL: String = `https://amritwprdev.piramalswasthya.org/commonapi-v1.0/`;
-    private _helpline1097BaseURL: String = `${IP1097}1097api-v1.0/`;
-    private _telephonyServerURL: String = `${telephonyServerIP}`;
+    private _commonBaseURL: String = environment.commonAPI;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)